packages/fastify-vite/plugin.mjs

```diff
@@ -86,11 +86,12 @@ export function viteFastify({ spa, clientModule } = {}) {
         fastify,
       }
 
-      jsonFilePath = join(
-        root,
-        findCommonPath(Object.values(fastify.outDirs)),
-        'config.json',
-      )
+      const commonDistFolder = findCommonPath(Object.values(fastify.outDirs))
```

Comment:
Is this function still needed? Wouldn't the `commonDistFolder` now always be equal to the value of `outDir` defined on line 14?

Reply:
I'll check!

Reply:
Did you conclude that running `findCommonPath` is still necessary? Under what circumstance would it be different from the `outDir` defined on line 14?

Reply:
It is in fact unnecessary now that the `outDir` of the client and SSR environments are built based on the global one. I'll do some more refinements there.